src/docs/asciidoc/index.adoc

```diff
@@ -1,8 +1,8 @@
 = Micronaut Gradle plugin
-:native-build-tools-version: 0.9.11
+:native-build-tools-version: 0.9.13
```

Comment:
I have created a separate PR for this: https://github.com/micronaut-projects/micronaut-gradle-plugin/pull/535